Submitted By:            Douglas Engstrand <matthew@linuxfromscratch.org>
Date:                    2019-05-31
Initial Package Version: 5.2.2
Upstream Status:         Not submitted
Origin:                  Douglas Engstrand
Description:             Changes EXTRAVERSION

diff --git a/Makefile b/Makefile
index 11358153d8f2..ec3706a66579 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 2
 SUBLEVEL = 0
-EXTRAVERSION = -rc2
+EXTRAVERSION = -thor_kernel
 NAME = Golden Lions
 
 # *DOCUMENTATION*
